jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -336,6 +336,9 @@ def _add_orientation(self, rotation_angle=None, east_left=None, label=None,
         if self.link_type_selected != 'WCS':
             raise ValueError("must be aligned by WCS to add orientation options")
 
+        if rotation_angle is not None:
+            self.rotation_angle = rotation_angle

```

Comment:
The `rotation_angle` traitlet isn't necessarily the currently set rotation, but rather the rotation that will be used if creating a new entry from the UI, right?  I'm not sure I'd expect a kwarg to `add_orientation` from the API to change that value then (and if it does, we probably should do the same for `east_left` and `label`?).

Can you please remind me what buggy behavior this is aiming to fix?  I am wondering if maybe the fix needs to be in footprints instead?